### PR TITLE
Fix the UtfGrid tile getJSON for loop

### PR DESCRIPTION
**Pull request by bwyss**
This PR fixes a small error in the for loop that processed the UtfGrid metadata information. With the fix in place the application should now load and respond to each GRV theme. To test you can compere the app's behavior with the behavior of a platform that does not have the fix.

### CODE REVIEW

File: openquakeplatform/openquakeplatform/templates/data_viewer_base.html [hunk 1]
```diff
@@ -29,8 +29,6 @@
     <link rel="stylesheet" href="{{ STATIC_URL }}css/d3.css" />
     <link rel="stylesheet" href="{{ STATIC_URL }}css/ng-table.min.css">
     <link rel="stylesheet" href="{{ STATIC_URL }}css/leaflet-dvf.css">
-    <link rel="stylesheet" href="{{ STATIC_URL }}bootstrap-extend/css/jasny-bootstrap.min.css">
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/templates/data_viewer_base.html]: Are you sure this dependency is not used by any other application which inherits this template?

Comment (bwyss) [inline openquakeplatform/openquakeplatform/templates/data_viewer_base.html]: no, that was a merge conflict, I will put it back

File: openquakeplatform/openquakeplatform/templates/data_viewer_base.html [hunk 2]
```diff
@@ -69,8 +68,6 @@
     <script src="{{ STATIC_URL }}js/base_map_options.js"></script>
     <script src="{{ STATIC_URL }}js/xml2json.js"></script>
     <script src="{{ STATIC_URL }}js/leaflet-dvf.js"></script>
-    <script src="{{ STATIC_URL }}bootstrap-extend/js/jasny-bootstrap.js"></script>
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/templates/data_viewer_base.html]: Please re-add also this javascript

**Comment by daniviga** [comment]
LGTM

**Pull request merged**